<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USBtinyISPGUIClass</class>
 <widget class="QMainWindow" name="USBtinyISPGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>387</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490000</width>
    <height>387000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>USBtinyISP GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icons/usbtinyisp-gui-48x48.png</normaloff>:/icons/usbtinyisp-gui-48x48.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Atmel Device:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="microcontrollerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Target AVR8 device</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="programmerTab">
       <attribute name="title">
        <string>&amp;Programmer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="actionGroupBox">
           <property name="title">
            <string>Action:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QRadioButton" name="eraseRadioButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Erase the flash memory of the target device.</string>
              </property>
              <property name="text">
               <string>Erase Memory</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="programRadioButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Program the flash memory or EEPROM of the selected device.</string>
              </property>
              <property name="text">
               <string>Program Memory</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="memoryComboBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Flash</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EEPROM</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="selectFileGroupBox">
           <property name="title">
            <string>Select File:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>HEX File:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="HEXFileLineEdit">
                <property name="minimumSize">
                 <size>
                  <width>290</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="openPushButton">
                <property name="minimumSize">
                 <size>
                  <width>128</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Select a program or EEPROM  file for flashing
the selected memory area of the target device </string>
                </property>
                <property name="text">
                 <string>&amp;Open</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>338</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="startPushButton">
                <property name="minimumSize">
                 <size>
                  <width>128</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Start with flashing of the selected memory area on the target device
or erase the flash memeory.</string>
                </property>
                <property name="text">
                 <string>&amp;Start</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>F&amp;uses Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="fusesLowByteGroupBox">
          <property name="title">
           <string>Fuses Low Byte</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="fusesHighByteGroupBox">
          <property name="title">
           <string>Fuses High Byte</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="defaultPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Set the values of the fuses back to the defaults from the configuration file.
No actual change on the device side.</string>
             </property>
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="readPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Read the values of the fuse bits from the selected device.</string>
             </property>
             <property name="text">
              <string>Read</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>189</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="writePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Write the given values for the fuses on the device.</string>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionDevice_Info"/>
    <addaction name="separator"/>
    <addaction name="actionConvertor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfigure">
   <property name="icon">
    <iconset>
     <normaloff>../../../icons/configure.png</normaloff>../../../icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../../../icons/exit.png</normaloff>../../../icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionConvertor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../icons/refresh.png</normaloff>../../../icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Convertor</string>
   </property>
  </action>
  <action name="actionDevice_Info">
   <property name="text">
    <string>Device Info</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>microcontrollerComboBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>memoryComboBox</tabstop>
  <tabstop>eraseRadioButton</tabstop>
  <tabstop>programRadioButton</tabstop>
  <tabstop>HEXFileLineEdit</tabstop>
  <tabstop>openPushButton</tabstop>
  <tabstop>startPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
